ir/type.def

```diff
@@ -879,6 +879,14 @@ class Type_Newtype : Type_Declaration, IAnnotated {
 #nodbprint
 }
 
+/// FIXME: Only required for P4-14. Remove.
+class Attribute : Declaration {
```

Comment:
why move this here from ir-v1.def?  That seems backwards as you'll get it built in to the compiler even when P4_14 is disabled?